kratos/python_scripts/assign_vector_variable_process.py

```diff
@@ -55,6 +54,9 @@ def __init__(self, Model, settings ):
                 for i in range(3):
                     settings["constrained"][i].SetBool(is_fixed)
 
+        if not settings.Has("value"):
```

Comment:
I think this makes sense

Reply:
Yeah, once again I opted for sticking to the informal standard.

Reply:
Added this to all other assign_vector processes

Reply:
those processes are super old, the standards have changed quite a bit over the years
Hence I would take the opportunity to throw proper errors (IMO)

What do others think?
@armingeiser ?